### ISSUE TITLE
CMD cd auto-complete fails 

### ISSUE THREAD

**Issue by adrianbrowning** [withfig/fig#538]
### Description: CD auto-complete fails
> Please include a detailed description of the issue (and an image or screen recording, if applicable)

![2021-10-23 12 37 17](https://user-images.githubusercontent.com/2430052/138554418-312b836b-02d1-4f27-bd33-297b02dfd72c.gif)

```
<img src="/autocomplete/v7/assets/loading_ellipsis.089306b1.gif" alt="Loading..." style="height: 20px;">
[Error] template suggestion did not work for template named: folders
	(anonymous function) (index.3be298e5.js:1:20195)
	asyncFunctionResume
	(anonymous function)
	promiseReactionJobWithoutPromise
	promiseReactionJob
[Error] Error: pty execute promise timed out. Running cmd + c in pseudo terminal just in case — index.3be298e5.js:1:4390
	(anonymous function) (index.3be298e5.js:1:20279)
	asyncFunctionResume
	(anonymous function)
	promiseReactionJobWithoutPromise
	promiseReactionJob
```

I have tried restarting my mac, restarting fig and re-installing fig. All to no avail  

 
### Details:
|macOS|Fig|Shell|
|-|-|-|
|10.15.7|Version 1.0.51 (B285)|/usr/local/bin/bash|
<details><summary><code>fig diagnostic</code></summary>
<p>
<pre>Version 1.0.51 (B285) [British - PC] 
UserShell: /usr/local/bin/bash
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /Users/adrianbrowning/.fig/bin/fig
Accessibility: true
Number of specs: 1
SSH Integration: false
Tmux Integration: true
Keybindings path: <none>
iTerm Integration: true 
Hyper Integration: false
VSCode Integration: true
Docker Integration: false
Symlinked dotfiles: false
Only insert on tab: false
Installation Script: true
PseudoTerminal Path: /Users/adrianbrowning/.cargo/bin:/Users/adrianbrowning/.rvm/gems/ruby-2.4.1/bin:/Users/adrianbrowning/.rvm/gems/ruby-2.4.1@global/bin:/Users/adrianbrowning/.rvm/rubies/ruby-2.4.1/bin:/Users/adrianbrowning:/bin:/usr/bin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Library/Apple/usr/bin:/Applications/Wireshark.app/Contents/MacOS:/Users/adrianbrowning/.fig/bin:/Users/adrianbrowning/Library/Python/2.7/bin/:/Users/adrianbrowning/.rvm/bin:/Users/adrianbrowning/.rvm/bin:/Users/adrianbrowning/Scripts/:/usr/local/opt/icu4c/bin:/usr/local/opt/icu4c/sbin:/usr/local/opt/terraform@0.11/bin:/usr/local/sbin:/usr/local/opt/awscli@1/bin:/Users/adrianbrowning/Documents/repos/txiki.js/build
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: ??? (???) - ???
Current working directory: ???
Current window identifier: ???</pre>
</p>
</details>

**Comment by AleksandarTokarev** [comment]
seeing same issue with ZSH + Macbook Pro 2018 with Iterm2

**Comment by mschrage** [comment]
Ugh sorry that the issue has popped back up @adrianbrowning and that you're running into it as well @AleksandarTokarev!

Fixing this bug is a top priority for me this week! Are either of you free to jump on a debugging call? (https://calendly.com/mschrage/30min)?

**Comment by AleksandarTokarev** [comment]
I don't know how, but after doing the stuff at https://forum.withfig.com/t/autocomplete-does-not-work-for-recommending-filepaths/39 and some closing/opening of Iterm2, it started to work, so i am not sure if i will be able to help. Will reach out to you if this pops up again @mschrage . Thanks a lot for the quick response!

**Comment by adrianbrowning** [comment]
Hi @mschrage I'm back from a week off, so I will find some time to do a debugging session with you.

**Comment by adrianbrowning** [comment]
So did an update today and got 
```
GLaDos:tmp:$ fig update

Pulling most up-to-date completion specs...
Run fig docs to learn how to contribute your own!

tar: specs: Not found in archive
tar: Error exit delayed from previous errors.
```

Using this: https://forum.withfig.com/t/figs-autocomplete-popup-is-not-showing-up/43
I found the readme at `~/.fig/autocomplete`
The readme was the only thing in the directory
So I followed the instructions the instructions listed, and the archive it pulls down doesn't contain any specs.

So went to : https://github.com/withfig/autocomplete/releases
Pulled down the latest specs.zip file
https://github.com/withfig/autocomplete/releases/download/spec-build-number-0.420.0/specs.zip
Put those into the directory, restarted, but still no luck :( 

Still get the error `template suggestion did not work for template named: folders`

**Comment by adrianbrowning** [comment]
Also get a 404 for https://unpkg.com/@withfig/autocomplete@2.39.0/build/artisan.js

**Comment by adrianbrowning** [comment]
Latest `fig diagnostic` report

```
Version 1.0.51 (B294) [British - PC] 
UserShell: /usr/local/bin/bash
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /Users/adrianbrowning/.fig/bin/fig
Accessibility: true
Number of specs: 255
SSH Integration: false
Tmux Integration: true
Keybindings path: <none>
iTerm Integration: true 
Hyper Integration: false
VSCode Integration: true
Docker Integration: false
Symlinked dotfiles: false
Only insert on tab: false
Installation Script: true
PseudoTerminal Path: /Users/adrianbrowning/.cargo/bin:/Users/adrianbrowning/.rvm/gems/ruby-2.4.1/bin:/Users/adrianbrowning/.rvm/gems/ruby-2.4.1@global/bin:/Users/adrianbrowning/.rvm/rubies/ruby-2.4.1/bin:/usr/local/opt/expat/bin:/Users/adrianbrowning:/bin:/usr/bin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Library/Apple/usr/bin:/Applications/Wireshark.app/Contents/MacOS:/Users/adrianbrowning/.fig/bin:/Users/adrianbrowning/Library/Python/2.7/bin/:/Users/adrianbrowning/.rvm/bin:/Users/adrianbrowning/Scripts/:/usr/local/opt/icu4c/bin:/usr/local/opt/icu4c/sbin:/usr/local/opt/terraform@0.11/bin:/usr/local/sbin:/usr/local/opt/awscli@1/bin:/Users/adrianbrowning/Documents/repos/txiki.js/build
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: /usr/local/Cellar/bash/5.1.4/bin/bash (99168) - ttys004
Current working directory: /Users/adrianbrowning/.fig/autocomplete
Current window identifier: 73764/% (com.apple.Terminal)
PATH: /Users/adrianbrowning/.rvm/gems/ruby-2.4.1/bin:/Users/adrianbrowning/.rvm/gems/ruby-2.4.1@global/bin:/Users/adrianbrowning/.rvm/rubies/ruby-2.4.1/bin:/usr/local/opt/expat/bin:/Users/adrianbrowning:/bin:/usr/bin:/usr/local/bin:/Users/adrianbrowning/.cargo/bin:/Users/adrianbrowning/.rvm/gems/ruby-2.4.1/bin:/Users/adrianbrowning/.rvm/gems/ruby-2.4.1@global/bin:/Users/adrianbrowning/.rvm/rubies/ruby-2.4.1/bin:/usr/local/opt/expat/bin:/Users/adrianbrowning:/bin:/usr/bin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Library/Apple/usr/bin:/Applications/Wireshark.app/Contents/MacOS:/Users/adrianbrowning/.fig/bin:/Users/adrianbrowning/Library/Python/2.7/bin/:/Users/adrianbrowning/.rvm/bin:/Users/adrianbrowning/Scripts/:/usr/local/opt/icu4c/bin:/usr/local/opt/icu4c/sbin:/usr/local/opt/terraform@0.11/bin:/usr/local/sbin:/usr/local/opt/awscli@1/bin:/Users/adrianbrowning/Documents/repos/txiki.js/build:/Users/adrianbrowning/.rvm/bin:/Users/adrianbrowning/Library/Python/2.7/bin/:/Users/adrianbrowning/Scripts/:/usr/local/opt/icu4c/bin:/usr/local/opt/icu4c/sbin:/usr/local/opt/terraform@0.11/bin:/usr/local/sbin:/usr/local/opt/awscli@1/bin:/Users/adrianbrowning/Documents/repos/txiki.js/build
FIG_INTEGRATION_VERSION: 5
```

**Comment by adrianbrowning** [comment]
I also tried doing simple commands to fig.pty.execute, like
`fig.pty.exectue("pwd", undefined, a=>console.log(a));`
`fig.pty.exectue("echo 'Hello World'", undefined, a=>console.log(a));`

Neither of these ever came back

**Comment by mschrage** [comment]
@adrianbrowning this has been fixed in v1.0.52! You can update by running `fig update` in your terminal. :)

**Comment by adrianbrowning** [comment]
@mschrage Confirming it's working! Awesome work, thank you!
